src/ucp/wireup/select.c

```diff
@@ -1271,6 +1305,10 @@ ucs_status_t ucp_wireup_select_lanes(ucp_ep_h ep, const ucp_ep_params_t *params,
         return status;
     }
 
+    if (need_am) {
```

Comment:
why need it here?

Reply:
removed it and resetting the value to `0` in `ucp_wireup_add_memaccess_lanes` function